docs/src/components/icons-list.astro

```diff
@@ -9,15 +9,33 @@ const icons = Object.keys(Icons) as (keyof typeof Icons)[];
 	{
 		icons.map((icon) => {
 			return (
-				<div class="icon-preview">
+				<button class="icon-preview" aria-label={`Copy ${icon} icon to clipboard`} id={icon}>
```

Comment:
Oops, comment here went missing.

We can use a `data-*` attribute instead of an `id` to store the icon name. Not a huge difference in practice, but in general I prefer using them for data like this we want to use.

```suggestion
				<button class="icon-preview" aria-label={`Copy ${icon} icon to clipboard`} data-name={icon}>
```